tasks.py

```diff
@@ -90,6 +90,12 @@ def adminserver(port=8001):
     cmd = '{} python manage.py runserver {} --nothreading'.format(env, port)
     run(cmd, echo=True, pty=True)
 
+# TODO - refactor? It is functional. e.g. 'invoke admin_task createsuperuser'
+@task
+def admin_task(args):
```

Comment:
Does `args` work? I wouldn't expect it to; invoke creates CLI options for each function argument, so I you would have to do

```python
inv admin_task -a createsuperuser
```

You might experiment with doing `*args` instead.